.prettierignore

```diff
@@ -1,3 +1,9 @@
+tmp/
+.github/
+
+# FIXME: Maybe we should lint/format the k8s files?
+kubernetes/
```

Comment:
I'll spin that out into an issue. It'd be good to do.

Reply:
We now have an issue for this here: https://github.com/NOAA-GSL/unified-graphics/issues/328